boards/arm/nrf9131ek_nrf9131/nrf9131ek_nrf9131_ns_defconfig

```diff
@@ -0,0 +1,35 @@
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_SOC_SERIES_NRF91X=y
+CONFIG_SOC_NRF9131_LACA=y
+CONFIG_BOARD_NRF9131EK_NRF9131_NS=y
+
+# Enable MPU
+CONFIG_ARM_MPU=y
+
+# Enable hardware stack protection
+CONFIG_HW_STACK_PROTECTION=y
+
+# Enable TrustZone-M
+CONFIG_ARM_TRUSTZONE_M=y
+
+# This Board implies building Non-Secure firmware
+CONFIG_TRUSTED_EXECUTION_NONSECURE=y
+
+# enable GPIO
+CONFIG_GPIO=y
+
+# Enable uart driver
+CONFIG_SERIAL=y
+
+# enable console
+CONFIG_CONSOLE=y
+CONFIG_UART_CONSOLE=y
+
+CONFIG_PINCTRL=y
+
+# enable PMIC
+CONFIG_I2C=y
+CONFIG_REGULATOR=y
+CONFIG_SENSOR=y
+CONFIG_NPM1300_CHARGER=y
```

Comment:
Why are these needed to be on by default for _ns builds?

Reply:
this is because of the charger function - it has to be configured for the hardware to work

Reply:
Because the board will brown out without that configuration. We cannot safely run firmware without the PMIC configured.